tests/test_remote_wrapper.py

```diff
@@ -28,7 +28,7 @@
 STD_TIMEOUT = 5  # Max time to send/receive a topic (seconds)
 LONG_TIMEOUT = 60  # Max Remote startup time (seconds)
 
-index_gen = utils.index_generator()
+index_gen = utils.index_generator(imin=1234)
```

Comment:
Please help to comment the fix here. I do not understand why the `imin=1234` fixes the issue. Thanks!